change/@ni-nimble-components-e81831d9-2db3-450d-b3a8-776ab5156e86.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Could we have some storybook cases that exercise this?
The select and combobox are using our old style where you need the funky json editor to test things out. Could you instead create some useful cases we can try out with options?
Some ideas for option sets: simple (which has a couple disabled items), long list, wide list

Reply:
Guess it would also need a fixed width, currently it seems to just grow